_minutes/2022-07-21-wecg.md

```diff
@@ -0,0 +1,141 @@
+ * [tomislav] The intention behind the changes in Firefox is that the popup (dark) theme should follow the one of the browser UI.
+ * [timothy] That's also what we do in Safari: it's supposed to match the browser's UI, not the web pages.
+ * [simeon] Next steps here is to get browser vendor comments on this issue.
+ * [tomislav] FYI on terminology: “chrome” (lower-case “c”) refers to the browser UI, not to be confused with Chrome (upper-case “C”), the web browser from Google.
+
+[Issue 244](https://github.com/w3c/webextensions/issues/244): Status labels for WECG GitHub issues
+
+ * [simeon] Crated a suggested initial set of "status" labels. Curious for feedback.
+ * [timothy] Carlos's label suggestions reflecting the state of consensus from each browser looks like a good addition to me.
+ * [tomislav] ES proposals have 4 stages. Something like that could be useful for our purposes. Mozilla has a [standard-positions Github repo](https://github.com/mozilla/standards-positions), with some standardized statuses (e.g. harmful, worth prototyping, etc. to reflect the support sentiment).
+ * [simeon] Would you be open to creating a half-pager or something like that?
+ * [tomislav] I'll write a comment on the issue.
+ * [rob] We now have three issues about labels. We should consolidate them:
+   * https://github.com/w3c/webextensions/issues/205
+   * https://github.com/w3c/webextensions/issues/233
+   * https://github.com/w3c/webextensions/issues/244 (the one being discussed now)
+ * [simeon] Based on initial feedback, we're not going to proceed with my suggestion here.
+ * [tomislav] Goal is to have consensus tracking.
+ * [rob] If we want to consolidate discussion, suggest using issue 233 as it's the most generic, non-prescriptive one.
+
+[Issue 12](https://github.com/w3c/webextensions/issues/12): allow to retrieve a frameID from an &lt;iframe> element
+
+ * [gaurang] This was raised a few weeks ago with a “Chrome: follow-up” label.
+ * [simeon] While reviewing issues with the label, it wasn't clear to me what exactly needs to be followed-up upon. Could you comment on the issue to clarify?
+ * [gaurang] I'll create a comment.
+ * [simeon] FYI Dave came up a while back to talk about the frameId proposal a while back.
+   * [rob] You meant documentId, right?
+   * [simeon] That's right, but I still need to follow up on that one.
+
+[Issue 162](https://github.com/w3c/webextensions/issues/162): DNR: disable individual static rules
+
+ * [felipe] Devlin [provided feedback on the current design document](https://docs.google.com/document/d/1NTOTr6iwm0dJbewWjnABmPo6h1QD1mKTpX60s6Klj-8/edit?disco=AAAAc3D154o); wanted to mention that here in case other parties are interested. Also wanted to encourage members to provide feedback on potential use cases for this capability. Delvin is asking if there are more specific use cases where extensions may leverage this capability, such as providing a user-facing control to disable individual static rules.
```

Comment:
```suggestion
 * [felipe] Devlin [provided feedback on the current design document](https://docs.google.com/document/d/1NTOTr6iwm0dJbewWjnABmPo6h1QD1mKTpX60s6Klj-8/edit?disco=AAAAc3D154o); wanted to mention that here in case other parties are interested. Also wanted to encourage members to provide feedback on potential use cases for this capability. Devlin is asking if there are more specific use cases where extensions may leverage this capability, such as providing a user-facing control to disable individual static rules.
```